src/org/labkey/test/components/ui/files/FileAttachmentContainer.java

```diff
@@ -195,6 +195,10 @@ protected class ElementCache extends Component<?>.ElementCache
         public Locator fileUploadScrollFooterLoc = Locator.tagWithClass("div", "file-upload__scroll-footer");
     }
 
+    public static String fileUploadIdentifier(String fieldIdentifier)
+    {
+        return fieldIdentifier + "-fileUpload"; // Issue 53394
+    }
```

Comment:
We should [revert](https://github.com/LabKey/testAutomation/pull/2473) to how this worked before the removal. This way it will work for anything that's passing in an un-encoded field name.
```suggestion
    /**
     * File upload fields append "-fileUpload" to the field's fieldKey
     * @param fieldIdentifier Identifier for the field; name ({@link String}) or fieldKey ({@link FieldKey})
     * @return FieldKey with expected suffix
     */
    public static FieldKey fileUploadFieldKey(CharSequence fieldIdentifier)
    {
        return FieldKey.fromFieldKey(FieldKey.fromName(fieldIdentifier) + "-fileUpload");
    }
```